### PR TITLE
Update list-datapartners.json

### PR DESCRIPTION
**Pull request by MeMent0L**
Change may be confirmed by Bruce, following email exchange.

### CODE REVIEW

**Comment by MeMent0L** [comment]
Modification pending confirmation by Bruce

**Review by frozer25 — APPROVED** [review]

File: list-datapartners.json [hunk 1]
```diff
@@ -46,5 +46,13 @@
...
     }
+  },
+  {
+  "name": "Innovation Atelier Ltd",
+  "accounts": ["0x12BD31628075C20919BA838b89F414241b8c4869"],
+  "links": {
+    "Home": "https://innovationatelier.com",
+    "Twitter": "https://twitter.com/@InnoAtelier"
   }
+}
```
Comment (oceanByte) [inline list-datapartners.json:57]: Syntax error

**Comment by alexcos20** [comment]
@brucepon  ?

**Comment by MeMent0L** [comment]
Fixed the identation, sorry for the fail at prettier

**Comment by alexcos20** [comment]
@MeMent0L - please run npm run lint first

**Comment by MeMent0L** [comment]
creating a new clean pull request

**Pull request closed without merging**